FAQ.md

```diff
@@ -0,0 +1,113 @@
+
+
+### How do USDp and synthetic currencies work?
+
+USDp and synthetic currencies within 10101 are referred to synthetics in finance.
+You will have a USD balance circulating on Lightning, holding nothing but sats and a short position combined.
+
+It's a financial product that hedges you against the volatility of Bitcoin and tracking the value of USD, using only Bitcoin.
+
+If you want to learn more about it we've produced a [blog post](https://10101.finance/blog/synthetic-stable) on this topic.
+
+### What's the point of the recovery phrase?
+
+Your recovery phrase (sometimes called "seed") is the only way to redeem your funds. If you've deleted the app or lost the data on your phone, you will not be able to access your funds without it. We will be unable to assist you either.
+
+You absolutely must save it and keep it in a safe place.
+
+In the case of 10101, this backup will enable you to restore your funds within the app from both your on-chain and lightning wallet, as well as your transaction history and open orders, if you haven't exceeded the time limit* for your futures contracts.
+
+### Do I have to run a node separately to use 10101?
+
+The lightning self-custodial wallet embeds a node directly on your device, so there's no need to run a separate node to use the app.
+
+As for the Bitcoin wallet (On-Chain), it will soon be possible to connect your own node, in order to be independent and sovereign in broadcasting and receiving your transactions from your device.
+
+### How can I fund my Lightning wallet?
+
+To fund your lightning wallet, you'll need to open a channel with 10101, with a minimum deposit size of 60,000 sats and a maximum of 3 million sats.
+
+Depending on the size of this channel, you'll be able to open larger or smaller positions in the trading section, or hold more or less synthetic currency in your wallet.
+
+### Can I fund my Lightning wallet with my On-chain balance?
+
+It is not yet possible to fund your Lightning with your on-chain balance directly from the app, you will need to fund it from another Lightning wallet, or use a third party service that will allow you to perform atomic swaps, such as [Boltz](https://boltz.exchange)
+
+### Can I open several positions at the same time?
+
+It is not yet possible to open several different positions. However, it is possible to resize your position at any time, without being able to change the leverage.
+
+### Why can I not trade with my full Lightning balance?
+
+You will not be able to open positions covering your entire Lightning balance, as you will need to keep a reserve on your balance.
+
+Part of this reserve must remain in your channel to keep it open. Another part is also reserved so that an On-chain transaction can be published, enabling you to recover your funds in any worst-case scenario.
```

Comment:
```suggestion
Part of this reserve must remain in your channel to keep it open. Another part is also reserved so that on-chain transaction fees can be covered, enabling you to recover your funds in any worst-case scenario.
```